libraries/AP_NavEKF2/AP_NavEKF2_core.h

```diff
@@ -50,6 +50,9 @@
 // target EKF update time step
 #define EKF_TARGET_DT 0.01f
 
+// mag fusion final reset altitude (using NED frame so altitude is negative)
```

Comment:
To avoid confusion I suggest that EKF2_MAG_FINAL_RESET_ALT is a positive value and that the negative sign is inserted in front of it where it is used on line 61